charts/ocis/values.yaml

```diff
@@ -4,7 +4,7 @@ image:
   # -- Image repository
   repository: owncloud/ocis
   # -- Image tag. Defaults to the chart's appVersion.
-  tag: ""
+  tag: "latest"
```

Comment:
is it possible to use tag `dev` after creating dev image `make -C ocis dev-docker`?
will it work?